### PR TITLE
Frontend/fix: #1264 HapticFeedback bugfix

### PR DESCRIPTION
**Pull request by Rohit4997**
## Type of Change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs [hunk 1]
```diff
@@ -1808,6 +1802,18 @@ getQuotesPolling pollingId action retryAction count duration push state = do
         _ <- pure $ updateLocalStage QuoteList
         doAff do liftEffect $ push $ action response
 
+startChat :: forall action. (action -> Effect Unit) -> HomeScreenState -> (String -> String -> String -> String -> action) -> action ->  action -> Flow GlobalState Unit
+startChat push initialState updateMessages openChatScreen initializeChat = do
+  when (not initialState.props.chatcallbackInitiated) do
+    void $ delay $ Milliseconds 500.0
+    _ <- doAff $ liftEffect $ storeCallBackMessageUpdated push initialState.data.driverInfoCardState.bppRideId "Customer" updateMessages
```
Comment (Vignesh-772) [inline Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs:1809]: use `liftFlow` instead `doAff $ liftEffect`

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs [hunk 2]
```diff
@@ -1808,6 +1802,18 @@ getQuotesPolling pollingId action retryAction count duration push state = do
...
+    _ <- doAff $ liftEffect $ storeCallBackOpenChatScreen push openChatScreen
```
Comment (Vignesh-772) [inline Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs:1810]: no need to discard the result use `void`

**Review by Vignesh-772 — CHANGES_REQUESTED** [review]

File: Frontend/ui-customer/src/Helpers/Utils.js [hunk 3]
```diff
@@ -522,7 +522,7 @@ export const contactPermission = function () {
 }
 
 export const performHapticFeedback = function () {
-  if(window.JBridge.performHapticFeedback){
+  if(window.JBridge.performHapticFeedback && (window.JBridge.getAndroidVersion >= 8) ){
```
Comment (MercyQueen) [inline Frontend/ui-customer/src/Helpers/Utils.js:525]: @Rohit4997 getAndroidVersion returns the SDK version of the device. So if you want to refer to android version 8 then here you should mention the SDK version which is 26 in this case

**Review by MercyQueen — CHANGES_REQUESTED** [review]

File: Frontend/ui-customer/src/Helpers/Utils.js (hunk 3 above)
Comment (Rohit4997) [reply Frontend/ui-customer/src/Helpers/Utils.js]: @MercyQueen  can you check now?

**Review by MercyQueen — APPROVED** [review]

**Review by witcher-shailesh — APPROVED** [review]

**Pull request merged**